src/inmanta/data/schema.py

```diff
@@ -19,25 +19,25 @@
 import logging
 import pkgutil
 import re
+from itertools import takewhile
 from types import ModuleType
-from typing import Any, Callable, Coroutine, List, Optional, Tuple
+from typing import Any, Callable, Coroutine, List, Optional, Set, Tuple
 
-from asyncpg import Connection, UndefinedTableError
-from asyncpg.transaction import Transaction
+from asyncpg import Connection, UndefinedColumnError, UndefinedTableError
+from asyncpg.protocol import Record
 
 # Name of core schema in the DB schema verions
 CORE_SCHEMA_NAME = "core"
 
 LOGGER = logging.getLogger(__name__)
 
-LEGACY_SCHEMA_VERSION_TABLE = "schemaversion"
 SCHEMA_VERSION_TABLE = "schemamanager"
 
 create_schemamanager = """
 -- Table: public.schemamanager
 CREATE TABLE IF NOT EXISTS public.schemamanager (
     name varchar PRIMARY KEY,
-    current_version integer NOT NULL
+    installed_versions integer[] NOT NULL
```

Comment:
I would have expected many rows on the same name, but I'm a bit of a dinosaur, I don't know if this has any disadvantage (probably not)

Reply:
I couldn't say either. This felt natural to me but I have extremely limited experience with conventions in this regard.

Reply:
The limitation of this approach is that the size of the array will be limited by the maximum field size of PostgreSQL, while the approach with many rows can grow without boundary. However, the field size limit is 1GB in PostgreSQL 13 ( I can't find the limit for PostgreSQL 10). With that limit we can go up to 250 million version. I think that would be enough.